CMakeLists.txt

```diff
@@ -2,20 +2,17 @@
 
 cmake_minimum_required (VERSION 2.8)
 cmake_policy (VERSION 2.8)
-project (noise C)
+project (io.elementary.music C)
 list (APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake)
 
 enable_testing()
 
-set (GETTEXT_PACKAGE ${CMAKE_PROJECT_NAME})
+set (GETTEXT_PACKAGE "noise")
 set (RELEASE_NAME "Luna")
 set (VERSION "0.4.2")
 set (VERSION_INFO "Stable Release")
 
 include (GNUInstallDirs)
-set (DATADIR ${CMAKE_INSTALL_PREFIX}/share)
```

Comment:
If you remove this line, translations are no longer loaded. Or if you really want to remove it you can replace `string langpack_dir = Path.build_filename (Build.DATADIR, "locale");` (line 114, in `Application.vala`) with `string langpack_dir = Path.build_filename ("/usr", "share", "locale");`